packages/indexer/src/sinks/vcr.ts

```diff
@@ -0,0 +1,16 @@
+import { Sink, type SinkWriteArgs } from "../sink";
```

Comment:
This sink should be in the `testing` module since it's never used outside of testing.